cardform/src/main/res/drawable/cf_icon_check.xml

```diff
@@ -5,6 +5,6 @@
     android:viewportHeight="8">
   <path
       android:pathData="M3.788,5.396L8.365,0.819l1.028,1.029 -5.605,5.605L0.607,4.272l1.028,-1.029z"
-      android:fillColor="#00A650"
+      android:fillColor="#007811"
       android:fillType="evenOdd"/>
```

Comment:
what do you think about adding color by config file?

Reply:
Done!